rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -42,57 +41,82 @@ def get_ppo_loss(fwd_in, fwd_out):
     return loss
 
 
+class PPOCatalog(Catalog):
```

Comment:
Love the idea of algorithm-specific catalogs and letting the user pass in an extended catalog. This is amazing in terms of giving maximum flexibility in overriding sub-algorithm pieces.